connector/connector_ldap.go

```diff
@@ -217,11 +238,74 @@ func (c *LDAPConnector) TrustedEmailProvider() bool {
 	return c.trustedEmailProvider
 }
 
+type LDAPPool struct {
+	m          sync.Mutex
+	conns      map[*ldap.Conn]*ldap.Conn
```

Comment:
I did consider to change this given that I only use the key of the map throughout the code. The value stored could be nil, so using a pointer type that uses the least amount of memory is probably the thing to look for.

Is your proposal the norm in Go for such usage of maps?